### ISSUE TITLE
ignoring toType: 'dir'

### ISSUE THREAD

**Issue by chrxs** [kevlened/copy-webpack-plugin#17]
Hi
I might be doing it wrong, and any help would be greatly appreciated,
I'm copy files and trying to get them to appear in a certain directory but it seems to be ignoring what I set,
This is my webpack config:
```js
var nodeModules = path.resolve(__dirname, '../node_modules');
new CopyWebpackPlugin([{
  from: nodeModules + '/angular-i18n/angular-locale*.js',
  to: 'angular/i18n',
  toType: 'dir'
}])
```

I want all the translation files to be in `dist/angular/i18n/`
but they appear in `dist/node_modules/angular-i18n/`

I realise I am setting the from to a file with an extension, but thought the toType would override this,
am I wrong?

The following does work, but copies all the files
```js
var nodeModules = path.resolve(__dirname, '../node_modules');
new CopyWebpackPlugin([{
  from: nodeModules + '/angular-i18n',
  to: 'angular/i18n',
  toType: 'dir'
}])
```

**Comment by kevlened** [comment]
Try this in the latest version:

    var nodeModules = path.resolve(__dirname, '../node_modules');
    new CopyWebpackPlugin([{
      context: nodeModules + '/angular-i18n',
      from: 'angular-locale*.js',
      to: 'angular/i18n'
    }]);

**Comment by uvathreya** [comment]
[webpack.dev.js.txt](https://github.com/webpack-contrib/copy-webpack-plugin/files/1918931/webpack.dev.js.txt)
Hi,
I ran into a similar problem. I kow that what I'm doing might not be correct as I'm new to Front end development. I've attached webpack-config.js file here. Request you to have a look at it and help me.

Problem: I'm trying to copy all the locale files from /node_modules/angular-i18n to the output dist folder in webpack. Some how the copied files are emitted. I'm also attaching the screenshot of the build . 
![image](https://user-images.githubusercontent.com/8099374/38859526-d5acdf1c-424b-11e8-88fe-ef0c0bf40cee.png)
